doc/man3/EVP_KEYMGMT.pod

```diff
@@ -0,0 +1,84 @@
+=pod
+
+=head1 NAME
+
+EVP_KEYMGMT,
+EVP_KEYMGMT_fetch,
+EVP_KEYMGMT_up_ref,
+EVP_KEYMGMT_free,
+EVP_KEYMGMT_provider
+- EVP key management routines
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ typedef struct evp_keymgmt_st EVP_KEYMGMT;
+
+ EVP_KEYMGMT *EVP_KEYMGMT_fetch(OPENSSL_CTX *ctx, const char *algorithm,
```

Comment:
My idea was to document all "fetch" functions in the same place. See the docs for `EVP_MD_fetch` and `EVP_CIPHER_fetch`

Reply:
Yeah, I noticed...  and quite frankly, I think that's not a good idea, as it separates one KEYMGMT function from the rest, and I'm thinking the same about the other fetch manuals.  I've just been too preoccupied to comment on that earlier.

What I *do* think is also that we make too little use of man7.  We could have a page about the ins and outs of fetching there, i.e. much of the prose in EVP_MD_fetch.pod, and then refer back to it from any page that describes the actual fetching function.

Reply:
I could live with that.

Reply:
I'll start that in another PR

Reply:
Just a reminder to start this other PR

Reply:
#9409